backuptar/tar.go

```diff
@@ -109,6 +109,69 @@ func BasicInfoHeader(name string, size int64, fileInfo *winio.FileBasicInfo) *ta
 	return hdr
 }
 
+// EncodeSDDLFromTarHeader reads the SDDL associated with the header of the current file
+// from the tar header and encodes it into a byte slice.
+func EncodeSDDLFromTarHeader(hdr *tar.Header) ([]byte, error) {
```

Comment:
This is really returning a security descriptor, not SDDL. So maybe `SecurityDescriptorFromTarHeader` would be better.

Reply:
Fixed.